plugins/nexus-repository-conda/src/main/java/org/sonatype/nexus/repository/conda/internal/hosted/Utils.java

```diff
@@ -0,0 +1,63 @@
+package org.sonatype.nexus.repository.conda.internal.hosted;
+
+import groovy.lang.Tuple2;
+import org.apache.commons.lang.StringUtils;
+import org.sonatype.nexus.common.collect.NestedAttributesMap;
+import org.sonatype.nexus.common.hash.HashAlgorithm;
+import org.sonatype.nexus.repository.conda.internal.hosted.metadata.PackageDesc;
+import org.sonatype.nexus.repository.storage.Asset;
+
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import static org.apache.commons.lang.StringUtils.isNotEmpty;
+
+public class Utils {
```

Comment:
`Utils` is a bit vague as a name for a class, would suggest something like "AttributeMorpher" or something that's more descriptive of what you are doing